soccer/gameplay/skills/capture.py

```diff
@@ -17,6 +18,8 @@ class Capture(single_robot_behavior.SingleRobotBehavior):
     CourseApproachDist = 0.4
     CourseApproachAvoidBall = 0.10
     DelayTime = .5
+    InterceptVelocityThresh = 0.5
```

Comment:
Thank you for not using magic numbers

Reply:
It might be a good idea to have these be changeable in the ui (adding things to ui is a great intro issue for next year)